bloptools/bayesian/transforms.py

```diff
@@ -0,0 +1,74 @@
+from typing import Union
+
+import botorch
+from botorch.acquisition.objective import PosteriorTransform
+from botorch.posteriors.gpytorch import GPyTorchPosterior
+from botorch.posteriors.posterior_list import PosteriorList  # pragma: no cover
+from torch import Tensor
+
+
+class TargetingPosteriorTransform(PosteriorTransform):
+    r"""An affine posterior transform for scalarizing multi-output posteriors."""
+
+    scalarize: bool = True
+
+    def __init__(self, weights: Tensor, targets: Tensor) -> None:
+        r"""
+        Args:
+            weights: A one-dimensional tensor with `m` elements representing the
+                linear weights on the outputs.
+            offset: An offset to be added to posterior mean.
```

Comment:
Let's fix the docstring style - looks like the class was copied over from another code base.